usage/launcher/src/test/java/org/apache/brooklyn/launcher/BrooklynWebServerTest.java

```diff
@@ -150,10 +151,9 @@ public void verifyHttpsCiphers() throws Exception {
         brooklynProperties.put(BrooklynWebConfig.TRANSPORT_CIPHERS, "XXX");
         try {
             verifyHttpsFromConfig(brooklynProperties);
-            fail("Expected to fail due to unsupported ciphers during connection negotiation");
+            fail("Expected to fail due to to unsupported ciphers during connection negotiation");
```

Comment:
@neykov tests pass for me too. Is your stack trace from the client or server side?